app/uk/gov/hmrc/brm/connectors/NRSConnector.scala

```diff
@@ -70,21 +68,11 @@ class NRSConnector(var httpPost: HttpPost = WSHttp, auditor: ScotlandAudit = new
       (detailsUri, Json.parse(
         s"""
            |{
-           | "$JSON_FIRSTNAME_PATH" : "${firstName(fName,aName)}",
+           | "$JSON_FIRSTNAME_PATH" : "${CommonUtil.forname(fName,aName)}",
            | "$JSON_LASTNAME_PATH" : "${NameFormat(lName)}",
            | "$JSON_DATEOFBIRTH_PATH" : "$dob"
            |}
          """.stripMargin))
   }
 
-
-  //add additional name to firstname while sending to respective service.
-  protected def firstName(firstName: String, additionalName: Option[String]): String = {
-    val forenames = BrmConfig.ignoreAdditionalName match {
-      case true => NameFormat(firstName)
-      case false => NameFormat(firstName).concat(" ").concat(NameFormat(additionalName.getOrElse(""))).trim
```

Comment:
@manishsw could be s"${NameFormat(firstName)} ${NameFormat(additionalName.getOrElse(""))}".trim